site/about2.html

```diff
@@ -0,0 +1,44 @@
+
+
+<html>
+
+	<link rel="stylesheet" href="style_about.css"
+
+	<head>
+		<title>The Digital Yogi</title>
+	</head>
+	
+	<body background="images/background pattern.png">
+		<nav><i>
+		  <a href="about2.html">About Me</a> |</i>
+		  <a href="port5.html">Portfolio</a> |
+		</nav>
+
+		<h1>GABRIELE GEIER</h1>
+
+		<hr> &nbsp;
+		
+		<div class="image1">
+			<img src="images/Gabby.jpg" border=2 height= 180 align= right>
+		</div>
+		
+			<h2>I AM A DIGITAL MARKETER AND LOVER OF YOGA!</h2>
+
+			<p>I am Gabriele Geier, a Digital Marketer for a non-profit in Washington, DC. I have 8 years experience in social media management, community engagement and marketing and graphic design. On the side, I love to practice yoga and travel!</p>
+
+		<br> </br>
```

Comment:
I know I said that HTML tags have beginnings and endings, but there are a couple of exceptions to that rule (like the ```<img>``` tag).  The ```<br>``` tag is one of those exceptions too.  There is nothing wrong with having the closing ```</br>``` tag here, but it does not really do anything since you cannot put any content or nest any other HTML tags inside a ```<br>``` tag.

A lot of times you will see ```<br />``` as a way of self-closing the tag, or you will just see ```<br>``` by itself.